src/Adapter/MSTest.Engine/PublicAPI/net/PublicAPI.Shipped.txt

```diff
@@ -1 +1 @@
-#nullable enable
```

Comment:
You probably run the script from windows powershell, and not from modern powershell, which adds bom, because old powershell utf8 means **with bom**, and in new utf8 means **without bom**.

So quickfix to not add more changes than necessary, run it from modern powershell.